_studio/hevce_hw/h265/include/mfx_h265_encode_hw.h

```diff
@@ -159,9 +159,8 @@ class MFXVideoENCODEH265_HW : public VideoENCODE
     mfxStatus CheckVideoParam(MfxVideoParam & par, ENCODE_CAPS_HEVC const & caps, bool bInit = false);
     mfxStatus Execute(mfxThreadTask task, mfxU32 uid_p, mfxU32 uid_a);
 
-    virtual mfxStatus ExtraCheckVideoParam(MfxVideoParam & /*par*/, ENCODE_CAPS_HEVC const & /*caps*/, bool bInit = false)
+    virtual mfxStatus ExtraCheckVideoParam(MfxVideoParam & /*par*/, ENCODE_CAPS_HEVC const & /*caps*/, bool /*bInit = false*/)
```

Comment:
I already wrote that I have a concern about our closed source Windows counterpart which shares this code: will this warning out on Windows compiler or not? I believe it will! If so, we may need to fix these warnings with (void)par in the function body.

Reply:
For windows ok (no warnings) follow variants:
1. void f(int /*a*/) {}
2. void f(int a){a;}
3. void f(int a) ((void)a;)
but for clang only 1 and 3